packages/mobile/src/components/TokenBottomSheet.tsx

```diff
@@ -102,6 +102,9 @@ function TokenBottomSheet({ isVisible, origin, onTokenSelected, onClose }: Props
     setPickerHeight(height)
   }
 
+  const maxHeight = Dimensions.get('window').height - 100
```

Comment:
can you pull up the `100` into a module-level or function-level variable with a descriptive name?